src/components/formikValidationSchema.ts

```diff
@@ -159,7 +159,7 @@ const validateFormik = <FormikValuesType>(
       }
       const testFunction = rules[ruleKey].test;
       if (testFunction) {
-        const testError = testFunction(value);
+        const testError = testFunction(values);
```

Comment:
Sikker på at vi ikke trenger å endre på noe mer for å gjøre dette?
Hvis du ser på test-funksjonen [her](https://github.com/NDLANO/editorial-frontend/blob/7a8b54cc5f387504b63207d3c30fdb764957babe/src/util/formHelper.js#L128:L130) så ser den ut til å forvente en liste med grepcodes feks?

Reply:
Skrev om ting til å bruke values.